9.6/paper.tex

```diff
@@ -312,9 +318,33 @@ \subsection{Advances in non-matching support}\label{sec:nonmatching}
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \subsection{Interface to the \trilinos \tpetra stack}\label{sec:tpetra}
 
+As a software project, \trilinos{} is designed as a stack of semi-independent
+packages building upon each other and working together. 
+As a foundation, \teuchos{} offers a set of basic tools,
+including parameter handling through XML files or command line interfaces 
+and the reference counted pointer \texttt{RCP} which predates 
+the smart pointer types introduced in \texttt{C++11}.
+The second layer in the stack is a package providing distributed linear algebra 
+classes for vectors and (sparse) matrices based on the \petra{}
+object model. The third layer is a set of interconnected packages
+providing preconditioners, various types of solvers 
+as well as more advanced techniques like automatic differentiation, 
+time integration and discretization.
+
+For now, there are two choices for the second layer 
+with \epetra{} providing \MPI{} only parallelism 
+and the newer \tpetra{} (templated Petra) providing additional shared memory 
+parallelism and GPU capabilities through \kokkos{}, referred to as \MPIx.
+Since the interface of \tpetra{} was overhauled and modernized 
+many of the packages in the third layer now have two generations as well.
+Therefore, the \MPI-only set of packages is also referred to as the 
+\epetra{} stack and the \MPIx{} packages form the \tpetra{} stack.
+As a consequence of the different interfaces, 
+a decision was made to deprecate the \epetra{} stack 
+and only provide basic maintenance until a full 
+removal from the repository in the course of 2025.
```

Comment:
We have to make clear that we are talking about `Trilinos`, not `deal.II` here.

Reply:
```suggestion
As a consequence of the different interfaces, 
a decision was made in the \trilinos project to deprecate the \epetra{} stack 
and only provide basic maintenance until a full 
removal from the \trilinos repository in the course of 2025.
```

Reply:
Agreed, better?

Reply:
Yeah